test/unit/remote_control_test.lua

```diff
@@ -530,9 +530,19 @@ function g.test_call()
         )
         t.assert_not(ok)
         t.assert_equals(type(err), 'cdata')
-        t.assert_equals(err.code, box.error.PROC_LUA)
         t.assert_equals(err.message, 'ScriptError')
 
+        -- Up to Tarantool 2.4 errors that come through net.box were
```

Comment:
Considering this comment https://github.com/tarantool/cartridge/issues/1058#issuecomment-699948095, I think it's better to remove lines that's not compatible with newest Tarantool versions. (It's a reason why I changed my opinion https://github.com/tarantool/cartridge/pull/1059#discussion_r494579391)

I think it's not useful to test that some tests don't fail on Tarantool **before** 2.4.

Reply:
Done